samples/excel/range/insert-delete-clear-range.yaml

```diff
@@ -6,90 +6,110 @@ name: 'Insert, delete, clear range'
 description: 'Explore the APIs to inset, delete and clear a range'
 script:
   content: |+
-    $("#setup").click(() => invoke(setup));
+    $("#setup").click(setup);
 
-    $("#insert-range").click(() => invoke(insertRange));
-    $("#delete-range").click(() => invoke(deleteRange));
-    $("#clear-range").click(() => invoke(clearRange));
+    $("#insert-range").click(insertRange);
+    $("#delete-range").click(deleteRange);
+    $("#clear-range").click(clearRange);
 
-    async function invoke(action: () => Promise<any>) {
+    async function setup() {
         try {
-            await action();
-        } catch (error) {
+            await Excel.run(async (context) => {
+                
+                await resetWorksheet(context.workbook.worksheets.getActiveWorksheet());
```

Comment:
Same comments as above -- don't re-double-fetch active worksheet.  have the former return it.